docs: add docs for editing permissions (#97)
GameFabric/docs: src/multiplayer-servers/getting-started/setup-third-party-oauth.md

You may check which group(s) a user belongs to from "Groups / Users" page (under "Access Management"). You may also add/remove user(s) from/to group(s) on the "Groups / Users" page.

> Navigate to the "Access Management" from the Sidebar Menu, and then into the "Roles" page. On the list that displays roles, find the role you would like to modify permissions of, and click "Edit".

Reviewer: ```suggestion
Navigate to "Access Management" from the Sidebar Menu, and then into the "Roles" page. In the list of roles, find the one you want to modify and click 'Edit'.
```

Reviewer: Just saw that this wasn't even part of your change. But should be adjusted anyways imo
